### PR TITLE
T-183 RS monitoring

### PR DESCRIPTION
**Pull request by matejscerba**

### CODE REVIEW

**Comment by michalpulpan** [comment]
@matejscerba 

**Backend:**
- backend function "get recommendations" is now returning some static data (so that we see functions are connected)


**Storefront:**
- `sendEvent`:
- - `src/storefront/pages/product/[id]/[slug].tsx` sends ` getRecommendations("PRODUCT_DETAIL"...`, `sendEvent("PRODUCT_DETAIL_ENTER",... ` and `sendEvent("PRODUCT_DETAIL_LEAVE",...`
- - `src/storefront/pages/order/[orderId]/completed.tsx` sends `sendEvent("ORDER", {...`
- - `src/storefront/components/ProductDetail/ProductVariants/Row.tsx` sends `sendEvent("PRODUCT_ADD_TO_CART", {`
- - `src/storefront/components/Common/ProductItem.tsx` sends `sendEvent("RECOMMENDATION_VIEW", {`

- `getRecommendations`:
- - `src/storefront/pages/index.tsx` calls `getRecommendations("HOMEPAGE", `
- - `src/storefront/pages/cart/index.tsx` calls `getRecommendations("CART",`
- - `src/storefront/pages/product/[id]/[slug].tsx` calls ` getRecommendations("PRODUCT_DETAIL"...`, `


Into both calls of category products backend method `GET /category/storefront/${id}/products` and `POST /category/storefront/${id}/products` is propagated parameter `recommender_session_id` in the URL.


PLS - rebase this branch to latests master, it's neccessary before we touch anything in the `CategoryDetailProductsStorefrontView` since there were some huge changes in sorting logic. Thanks 💯

File: src/backend/core/core/settings.py [hunk 1]
```diff
@@ -112,10 +112,10 @@
     # }
     "default": {
         "ENGINE": "django.db.backends.postgresql_psycopg2",
-        "NAME": os.environ.get("POSTGRES_DB", ""),
-        "USER": os.environ.get("POSTGRES_USER", ""),
-        "PASSWORD": os.environ.get("POSTGRES_PASSWORD", ""),
-        "HOST": os.environ.get("POSTGRES_HOST", ""),
+        "NAME": os.environ.get("POSTGRES_DB", "postgres"),
```
Comment (michalpulpan) [inline src/backend/core/core/settings.py:115]: Could you please set this aj "ecoseller"? Since initial DB in our `docker-compose`is `ecoseller`

**Review by michalpulpan — APPROVED** [review]
Very briefly went over non-recommender parts of `backend`, `storefront` and `dashboard` and it looks ok 😂 . I have no clue about rest of the code, but you have my full trust 🙏 😂

**Review by lakatop — APPROVED** [review]
merge pls

**Comment by lakatop** [comment]
image

File: src/backend/core/core/settings.py (hunk 1 above)
Comment (matejscerba) [reply src/backend/core/core/settings.py]: Done

**Pull request merged**